src/scrapers/base-isracard-amex.ts

```diff
@@ -204,9 +204,9 @@ function convertTransactions(txns: ScrapedTransaction[], processedDate: string):
   });
 }
 
-async function fetchTransactions(page: Page, options: ExtendedScraperOptions, startMoment: Moment, monthMoment: Moment): Promise<ScrapedAccountsWithIndex> {
-  const accounts = await fetchAccounts(page, options.servicesUrl, monthMoment);
-  const dataUrl = getTransactionsUrl(options.servicesUrl, monthMoment);
+async function fetchTransactions(page: Page, options: ScraperOptions, companyServiceOptions:CompanyServiceOptions, startMoment: Moment, monthMoment: Moment): Promise<ScrapedAccountsWithIndex> {
+  const accounts = await fetchAccounts(page, companyServiceOptions.servicesUrl, monthMoment);
+  const dataUrl = getTransactionsUrl(companyServiceOptions.servicesUrl, monthMoment);
```

Comment:
Do you have to split the `companyServiceOptions` from `options`?

Reply:
These fields are not `ScraperOptions` but amex/isracard specific values that are internal to this file - defined inside `fetchData` and merged with the scraper options.